src/Compute/Compute/help/Set-AzVmssDiskEncryptionExtension.md

```diff
@@ -257,7 +272,7 @@ Accept wildcard characters: False
 ```
 
 ### -VolumeType
-Type of the volume (OS or Data) to perform encryption operation
+Type of the volume (OS or Data) on which to perform encryption operation. This parameter is required when encrypting Linux virtual machines, and must be set to "Data". This parameter can be omitted when encrypting Windows virtual machines.
```

Comment:
If it makes it more clear to reader why it must be set to Data for Linux.  OS disk encryption for VMSS is supported on Windows, but is not yet supported on Linux.